.circleci/config.yml

```diff
@@ -32,8 +52,16 @@ workflows:
       - circleci-maven-release-orb/run-maven-release:
           requires:
             - approve-release
-          ssh-fingerprints: "25:1d:9f:0d:ac:7a:0e:19:e3:1c:25:53:d2:cb:13:cc"
-          context: rso-base
-          filters:
-            branches:
-              only: main
+          <<: *release-args
+
+# uncomment to enable automatic release from any commit to 'main' branch
```

Comment:
Ooh I like this. Keep it commented for now, but I'll work on the courage to enable this and give it a rip :)